tests/cases/fourslash/findReferencesBindingPatternInJsdocNoCrash1.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference path='fourslash.ts' />
+
+// @moduleResolution: node
+
+// @Filename: node_modules/use-query/package.json
+////{
+////  "name": "use-query",
+////  "types": "index.d.ts"
+////}
+// @Filename: node_modules/use-query/index.d.ts
+////declare function useQuery(): {
+////  data: string[];
+////};
+
+// @Filename: node_modules/use-query/package.json
```

Comment:
yes, it should - the package split shouldn't be required for this to be reproduced at all, it was just faster for me to keep the package structure as-is (like in the original real-world code)

Reply:
If u feel strongly about it, I can simplify this further or add an extra test case. I need to refresh myself on the whole situation (which I'll do soon since I plan to finish this PR) - but the file split itself might have been important for this (I'm not 100% sure about it right now though)